Rakefile

```diff
@@ -80,7 +80,7 @@ namespace :test do
     missing.reject! { |f| ! File.file? f }
     missing.reject! { |f| f =~ %r{test/(integration|cookbooks)} }
     missing.reject! { |f| f =~ %r{test/unit/mock} }
-    missing.reject! { |f| f =~ %r{test.*helper} }
+    missing.reject! { |f| f =~ /test.*helper/ }
```

Comment:
This was chefstyle -a being applied.  I imagine in this case it felt the %r was not needed because there are no path separators.... 

and yes, I just checked, it does change it back to that, //.

Reply:
What I was asking is how this PR needed this at all? Master should be clean wrt chefstyle, so how did this come about?